src/firefly/js/templates/lightcurve/PTF/PTFMissionOptions.js

```diff
@@ -0,0 +1,292 @@
+            [LC.META_TIME_CNAME]: Object.assign(getTypeData(LC.META_TIME_CNAME, '',
+                'time column name',
+                'Time Column:', labelWidth),
+                {validator: null}),
+            [LC.META_FLUX_CNAME]: Object.assign(getTypeData(LC.META_FLUX_CNAME, '',
+                'value column name',
+                'Value Column:', labelWidth),
+                {validator: null}),
+            [LC.META_TIME_NAMES]: Object.assign(getTypeData(LC.META_TIME_NAMES, '',
+                'time column suggestion'),
+                {validator: null}),
+            [LC.META_FLUX_NAMES]: Object.assign(getTypeData(LC.META_FLUX_NAMES, '',
+                'value column suggestion'),
+                {validator: null}),
+            ['cutoutSize']: Object.assign(getTypeData('cutoutSize', '',
+                'image cutout size',
+                'Cutout Size (arcmin):', 100)),
+            [LC.META_ERR_CNAME]: Object.assign(getTypeData(LC.META_ERR_CNAME, '',
+                'value error column name',
+                'Error Column:', labelWidth))
+        };
+
+        var defV = Object.assign({}, defValues);
+
+        const missionKeys = [LC.META_TIME_CNAME, LC.META_FLUX_CNAME, LC.META_URL_CNAME];
+        const missionListKeys = [LC.META_TIME_NAMES, LC.META_FLUX_NAMES];
+        const validators = getFieldValidators(missionEntries);
+
+        missionListKeys.forEach((key) => {
+            set(defV, [key, 'value'], get(missionEntries, key, []));
+        });
+
+        // set value and validator
+        missionKeys.forEach((key) => {
+            set(defV, [key, 'value'], get(missionEntries, key, ''));
+            if (has(validators, key)) {
+                set(defV, [key, 'validator'], validators[key]);
+            }
+        });
+        Object.keys(generalEntries).forEach((key) => {
+            set(defV, [key, 'value'], get(generalEntries, key, ''));
+        });
+        return defV;
+    };
+};
+
+
+function getFieldValidators(missionEntries) {
+    const fldsWithValidators = [
+        {key: LC.META_TIME_CNAME, vkey: LC.META_TIME_NAMES},
+        {key: LC.META_FLUX_CNAME, vkey: LC.META_FLUX_NAMES},
+        {key: LC.META_URL_CNAME}
+
+    ];
+    return fldsWithValidators.reduce((all, fld) => {
+        all[fld.key] =
+            (val) => {
+                let retVal = {valid: true, message: ''};
+                const cols = get(missionEntries, fld.vkey, []);
+                if (cols.length !== 0 && !cols.includes(val)) {
+                    retVal = {valid: false, message: `${val} is not a valid column name`};
+                }
+                return retVal;
+            };
+        return all;
+    }, {});
+}
+
+
+/**
+ * Pregex pattern for wise, at least to find mjd and w1mpro if present
+ * @type {string[]}
+ */
+const xyColPattern = ['\\w*jd\\w*', 'w[1-4]mpro\\w*'];
+export function ptfOnNewRawTable(rawTable, missionEntries, generalEntries, converterData, layoutInfo) {
```

Comment:
@wmiipac for ticket IRSA-514, you only need to care about `PTFPlotRequests.js`. Why do you care about `PTFMissionOptions.js` ? You shouldn't, right?
I agree that the request will be different of course but is fair to add code as part of the PR so you get a code to change only ;-)